src/rgw/rgw_op.h

```diff
@@ -45,6 +45,7 @@
 #include "rgw_lc.h"
 #include "rgw_torrent.h"
 #include "rgw_tag.h"
+#include "rgw_cksum.h"
```

Comment:
the only other addition to this file is `string cksum;` so i don't think this include is needed

Reply:
true, it's all in the .cc now